modular_skyrat/modules/modular_items/lewd_items/code/lewd_helpers/pref_checking.dm

```diff
@@ -33,3 +33,28 @@
 	log_message(message_to_log + ".", LOG_GAME)
 
 	return FALSE
+
+
+/// Checks the prefs of all mobs in `view()`. If there is a mob with the `pref_to_check` set to false, we return `FALSE` otherwise, we return `TRUE`
+/mob/living/proc/check_sight_prefs(datum/preference/toggle/pref_to_check)
```

Comment:
```suggestion
/mob/living/proc/check_prefs_in_view(datum/preference/toggle/pref_to_check)
```
Not sure how to best name this without getting overly verbose, the above is only a suggestion. check_audience_prefs sounds too general, not clear that it checks local audience only.

Reply:
check_prefs_in_view fits the scheme we use on others